src/Esprima/JavascriptParser.cs

```diff
@@ -2074,6 +2078,10 @@ private UnaryExpression ParseBasicUnaryExpression()
         {
             TolerateError(Messages.StrictDelete);
         }
+        if (_context.Strict && unaryExpr.Operator == UnaryOperator.Delete && unaryExpr.Argument is MemberExpression m && m.ChildNodes.Last() is PrivateIdentifier)
```

Comment:
Enumeration (`m.ChildNodes.Last()`) looks unnecessary here. Wouldn't `m.Property` be enough?

Reply:
think so